### PR TITLE
rerender table view when file is uploaded or deleted

### PR DESCRIPTION
**Pull request by dudziema**
Implemented feature which show new row in table when file is uploaded and delete row in table when file is deleted by user.

### CODE REVIEW

File: src/services/WebSocketService.js [hunk 1]
```diff
@@ -120,7 +117,7 @@ webSocketsService.install = function (Vue) {
   Vue.config.globalProperties.$addWsOnMessageListener = function (
     listenerFunction
   ) {
-    wsOnmessageListeners = listenerFunction; //.push(listenerFunction);
+    wsOnmessageListeners = listenerFunction;
```
Comment (serek8) [inline src/services/WebSocketService.js]: Can you make `wsOnmessageListeners` an array and then push `listenerFunction` to it? In this case, you can add many listeners

File: src/views/PhoneFiles.vue [hunk 2]
```diff
@@ -67,10 +68,12 @@ export default {
 
     const dropHandler = (ev) => {
       this.$dropHandler2(ev);
```
Comment (serek8) [inline src/views/PhoneFiles.vue]: `dropHandler2` Why **2**?

File: src/views/PhoneFiles.vue [hunk 3]
```diff
@@ -67,10 +68,12 @@ export default {
...
+      
     };
 
     const selectedFile = () => {
       dropZoneFile.value = document.querySelector(".dropzoneFile").files[0];
+
```
Comment (serek8) [inline src/views/PhoneFiles.vue]: Remove new line

**Review by serek8 — CHANGES_REQUESTED** [review]

**Review by serek8 — APPROVED** [review]

File: src/views/PhoneFiles.vue (hunk 2 above)
Comment (dudziema) [reply src/views/PhoneFiles.vue]: corrected in next PR

**Pull request merged**